Readme.md

```diff
@@ -1,6 +1,24 @@
 [![Express Logo](https://i.cloudup.com/zfY6lL7eFa-3000x3000.png)](http://expressjs.com/)
 
-  Fast, unopinionated, minimalist web framework for [Node.js](http://nodejs.org).
+  **Fast, unopinionated, minimalist web framework for [Node.js](http://nodejs.org).**
+
+  **This project has a [Code of Conduct][].**
+
+  ## Table of contents
```

Comment:
Let's drop this to an h3, github's CSS is giving these list items margin w/ the h2, and I think an h3 would fix that
```suggestion
  ### Table of contents
```

Reply:
hmm no that's not it, weird, what is adding the extra margin?

Reply:
It's something to do with the indentation happening above the header, gonna push a commit to fix it

Reply:
ah I can't push to your branch, if you check "Contributors can push" then I should be able to.

Reply:
permission seems to be granted, thank you very much for your suggestions

Reply:
pushed up some changes